runelite-client/src/main/java/net/runelite/client/ui/IconTextField.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2017, Adam <Adam@sigterm.info>
```

Comment:
Why is Adam's name on this one? You wrote it

Reply:
He had written the previous version, I rewrote it and moved it to another folder.